website/profile/views.py

```diff
@@ -44,22 +42,14 @@
 logger = logging.getLogger(__name__)
 
 
-def date_or_none(date):
```

Comment:
Ah, so you removed the test because you removed this code, because it was unused?